build.gradle

```diff
@@ -198,10 +212,6 @@ tasks.register('manifests', Copy) {
     into("build/sample-manifests")
 }
 
-tasks.register('integrationTest', Test) {
```

Comment:
This is actually not a duplicate of line 122, this is part of the parent, the other is in a subprojects block and applied to each subproject